components/FlexibleHeader/src/MDCFlexibleHeaderView.m

```diff
@@ -27,11 +27,11 @@
 #endif
 
 // The maximum default opacity of the shadow.
-static const float kDefaultVisibleShadowOpacity = 0.4f;
+static const float kDefaultVisibleShadowOpacity = (float)0.4;
```

Comment:
does this and the static floats below also need to be changed? I thought .f = float

Reply:
I'd like to keep floats as (float)0.4 throughout the code base. 
1. It means we can consistently avoid the `0.4f` syntax we decided to drop.
2. I think it (subjectively) indicates that the author carefully thought about the width of the value and decided it should explicitly be single-width. I suspect that much of the desire to write `0.4f`came from an aversion to typing more characters and not an intention to drop the precision that 64-bit values afford.